src/state/connection-mixin.ts

```diff
@@ -97,6 +91,12 @@ export const connectionMixin = <T extends Constructor<HassBaseEl>>(
                 err
               );
             }
+            if (
+              err.error.code === ERR_CONNECTION_LOST &&
+              serviceCallWillDisconnect(domain, service)
+            ) {
+              return undefined;
```

Comment:
Not sure what to do about the `lint` error

Reply:
You should update the return type in <https://github.com/home-assistant/frontend/blob/dev/src/types.ts#L226>